tests/unit_test/app_common/workflow/global_statistics_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.abstract.statistics_def import MetricConfig
+from nvflare.app_common.app_constant import StatisticsConstants as SC
+
+from .mock_global_statistics import MockGlobalStatistics
+
+
+class TestGlobalStatistics:
+    @classmethod
+    def setup_class(cls):
+        print("starting class: {} execution".format(cls.__name__))
+        metric_configs = {
+            "count": {},
+            "mean": {},
+            "sum": {},
+            "stddev": {},
+            "histogram": {"*": {"bins": 10}, "Age": {"bins": 5, "range": [0, 120]}},
+        }
+        cls.stats_controller = MockGlobalStatistics(metric_configs=metric_configs, writer_id="")
```

Comment:
Do you need the class `MockGlobalStatistics`?
We could just put `GlobalStatistics` here?

Reply:
For the method you are testing here, you don't need to provide FLContext right?